source_type_handlers.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/RedHatInsights/sources-api-go/dao"
+	"github.com/RedHatInsights/sources-api-go/util"
+	"github.com/labstack/echo/v4"
+)
+
+// function that defines how we get the dao - default implementation below.
+var getSourceTypeDao func(c echo.Context) (dao.SourceTypeDao, error)
+
+func getSourceTypeWithoutTenant(_ echo.Context) (dao.SourceTypeDao, error) {
+	// we do not need tenancy for source type.
+
+	return &dao.SourceTypeDaoImpl{}, nil
+}
+
+func SourceTypeList(c echo.Context) error {
+	SourceTypeDB, err := getSourceTypeDao(c)
```

Comment:
Can we make these lowercase throughout this file? Uppercase tends to indicate that it is exported from the package.